docs/tutorials/mastering.md

```diff
@@ -30,31 +13,621 @@ For the purpose of this guide, the following values will be used.
 | `core`       | 10.203.0.0/24 | 10.203.0.1 |
 | `ran`        | 10.204.0.0/24 | 10.204.0.1 |
 
-### IP Addresses
 
+## Prepare Host
+
+Create local network bridges to be used by LXD later.
```

Comment:
Is this being run on the host?  We cannot assume enp4s0 is everyone's NIC, and that they are using DHCP.  Instead, we should create a new netplan yaml (like `/etc/netplan/50-sdcore-networks.yaml`) that contains only what we need (bridges, etc)

Reply:
We use a netplan yaml which has a custom name. I agree. The interface name and DHCP configuration will be special to user environment. I added a comment to customize those settings according to requirement.

Reply:
Done